pkg/pillar/cmd/zedmanager/memorysizemgmt.go

```diff
@@ -40,6 +43,18 @@ func getRemainingMemory(ctxPtr *zedmanagerContext) (uint64, uint64, uint64, erro
 		}
 	}
 	memoryReservedForEve := ctxPtr.globalConfig.GlobalValueInt(types.EveMemoryLimitInBytes)
+	if vault.ReadPersistType() == types.PersistZFS {
+		byteCountStr, err := ioutil.ReadFile("/sys/module/zfs/parameters/zfs_arc_max")
+		if err != nil {
+			log.Errorf("failed to get data from zfs_arc_min. value: %s; code: %v", byteCountStr, err)
+		}
```

Comment:
shouldn't this be an else branch since you drive along after the error from ReadFile?

Reply:
Initially, I did not consider this a critical event. But I corrected, and added an error return if reading the file ended with an error.

Reply:
I wasn't commenting on that but that if you hit that first error you're also hit the second since Atoi will be called on an empty string. I don't know if the file always exists nor what the implications returning an error as opposed to picking the default value. Is that a safe?

FWIW /sys/module/zfs/parameters/zfs_arc_max doesn't exist on a EVE device not runnig with ZFS so I don't know exactly when it is created.

Reply:
@eriknordmark that is not an issue since that code is under condition for ZFS type, look at line 46.

Reply:
>Atoi will be called on an empty string
it wouldn't, because there is a `return` under this if. Code execution path will stop here.